docs/tutorial/package.md

```diff
@@ -146,69 +146,16 @@ def load():
 
 ## Modify the README
 
-Let's change the README. By default it's a file `README.rst`.
-
-Let's change it to `README.md`. So, change the extension from `.rst` to `.md`.
-
-So that we can use Markdown instead of reStructuredText.
+The default README has a `.md` extension and supports Markdown syntax. It can be replaced with an `.rst` file (reStructuredText), along with the corresponding change in `pyproject.toml`'s project metadata.
 
-And change the file to have something like:
+Let's change the README to have something like:
 
 ```Markdown
 # Portal Gun
 
 The awesome Portal Gun
 ```
 
-## Modify your project metadata
```

Comment:
I think this whole section can be deleted, as the project metadata doesn't require updating anymore.

Reply:
I agree.